apps/openchallenges/app/src/config/config.json.template

```diff
@@ -6,5 +6,6 @@
   "environment": "${ENVIRONMENT}",
   "googleTagManagerId": "${GOOGLE_TAG_MANAGER_ID}",
   "ssrApiUrl": "${SSR_API_URL}",
-  "enableOperationFilter": "${ENABLE_OPERATION_FILTER}"
+  "enableOperationFilter": "${ENABLE_OPERATION_FILTER}",
+  "showAnnouncement": "${SHOW_ANNOUNCEMENT}"
```

Comment:
TO-DO: test if removing double quote will resolve this issue

Reply:
Tested by using unquoted value e.g. `showAnnouncement: ${SHOW_ANNOUNCEMENT}` in the template, and the type is still string - see the testing console logs:
```
Raw showAnnouncement from config: false
typeof showAnnouncement: string
```

It looks like `envsubst` always performs string substitution, regardless of quoting